api/src/test/resources/io/kaoto/backend/api/resource/amq-amq.yaml

```diff
@@ -1,5 +1,5 @@
 - from:
-    uri: activemq:null:queue
```

Comment:
Made this test strong while debugging to check what happened with the path parameters.